### DISCUSSION TITLE
Attuatore IR RF con Tuya Smart Life app

### DISCUSSION THREAD

**Discussion by dertorquy** [indomus/forum#2679] [Hardware & Firmware]
Ciao a tutti, primo post su questo forum, complimenti per il sito indomus.
Ho letto un po' del sito, volevo chiedere: vorrei comprare un attuatore infrarosso e radio e leggo che Broadlink fa ottimi prodotti. C'è qualche possibilità di farli girare con la app in oggetto, magari mettondoci qualcosa in mezzo (tipo forse ifttt, ma leggevo che ci vuole comunque una app aggiuntiva che non è ideale)?
Altrimenti mi sapete consigliare un device compatibile diretto con Tuya (mi serve IR e radio codice fisso, meglio anche rolling).
Grazie mille
(Per curiosità: Tuya app è un Hub personale?)

**Comment by KC-inDomus** [comment]
Tuya è un ecosistema che funge da HUB, ma non "personale", bensì chiuso su se stesso.
La cosa assolutamente migliore è usare [Broadlink via Home Assistant](https://indomus.it/formazione/come-usare-gli-attuatori-broadlink-su-home-assistant/).

**Reply by dertorquy** [reply]
Grazie KC per il riscontro.
Ho visto che la guida di HA su inDomus è super esaustiva, ti posso fare solo una domanda, prima di mettermi a leggere tutto? Il vantaggio di HA è che posso usare varie marche e tecnologie di devices, però per esempio per comandare una lampadina Philips devo comunque prima installare sul telefono o sul PC la app di Philips, collegare la lampadina a tale app e poi integrare tale app in HA, corretto? Cioè non posso saltare il passaggio app proprietaria.
Grazie

**Reply by netdiver** [reply]
Nello spirito della community, provo a risponderti io, ma @KC-inDomus mi saprà correggere.
La risposta è "Dipende".
Ci sono alcuni dispositivi a cui si può cambiare il firmware per farli funzionare "cloud-free" (ad esempio i dispositivi basati su ESP8266 o ESP32 che possono caricare [Tasmota ](https://indomus.it/componenti/sonoff-tasmota/) o [ESPHome](https://indomus.it/componenti/esphome/)).
Ci sono altri dispositivi che sono stati reverse engineered o che espongono una API per poterli gestire senza passare dal cloud del produttore. Anche la lampadina Philips che citi ha questa possibilità, anche se... vedi [https://www.home-assistant.io/blog/2023/09/22/philips-hue-force-users-upload-data-to-cloud/](https://www.home-assistant.io/blog/2023/09/22/philips-hue-force-users-upload-data-to-cloud/).
Altri dispositivi, tipo il mio Condizionatore Hitachi, ancora non sono stati "liberati" dal cloud ufficiale del produttore.

L.

**Reply by KC-inDomus** [reply]
rispondendo alla tua domanda diretta, un HUB personale per lo più ti sgancia dal giogo dell'app del produttore, poi il caso dell'integrazione [local o cloud](https://indomus.it/formazione/integrazioni-local-cloud-e-stati-via-polling-push-come-la-domotica-dialoga-coi-componenti/) dipende da caso a caso. Broadlink per esempio è 100% locale

**Reply by dertorquy** [reply]
Grazie mille a tutti per i chiarimenti, molto utile. Aventi tutto su Smart Life, adesso valuto se passare a HA, mi piacerebbe molto perché è sempre un casino trovare devices di qualità elevata che sono compatibili Smart Life...

**Reply by KC-inDomus** [reply]
con HA ti si aprirà un mondo.
